apps/web/app/[locale]/kanban/page.tsx

```diff
@@ -36,116 +33,144 @@ const Kanban = () => {
 
 	const teamMembers: IImageOverlapper[] = [];
 
-	activeTeamMembers.map((member: any)=> {
+	activeTeamMembers.map((member: any) => {
 		teamMembers.push({
 			id: member.employee.user.id,
 			url: member.employee.user.imageUrl,
 			alt: member.employee.user.firstName
-		})
+		});
 	});
+	console.log('data-kanban', data);
```

Comment:
@Anishali2 clean up "console.log" in PRs before set it for review please :)

Reply:
Sorry for wasting you time on figuring out small things 
It will not happen in the future